### DISCUSSION TITLE
How to search issues having referenced pull request

### DISCUSSION THREAD

**Discussion by fenuks** [community/community#22163] [New to GitHub]
<p>After reading <a href="https://help.github.com/articles/searching-issues-and-pull-requests/" title="help page url" rel="nofollow noopener">help page</a> on advanced search I still wasn’t able to achieve what I aim for. Namely, I would like to search for all issues that were referenced by pull request. Gladly, with option to filter by pull request state in order to find all issues that are no longer relevant, like <a href="https://github.com/w0rp/ale/issues/1362" title="issue with closed pull request" rel="nofollow noopener">this one</a>.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/fenuks">@fenuks</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to Use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Cheers!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/fenuks">@fenuks</a>,</p>
<p>It’s not currently possible to search based on the filters you are describing. The full list of supported filters for GitHub search are at the link you already mentioned:</p>
<p><a href="https://help.github.com/articles/searching-issues-and-pull-requests/" rel="nofollow noopener">https://help.github.com/articles/searching-issues-and-pull-requests/</a></p>
<p>That said, we’re always working to improve GitHub, and we consider every suggestion we receive. I’ve logged your feedback in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by fenuks** [comment]
<p>Hello <a class="mention" href="/nadiajoyce">@nadiajoyce</a>, and thank you for your kind reply. It’s the only suggestion I have at the moment, albeit immodestly I think it’s good one :-), and hoping that GitHub issues filtering will be one day enhanced to support this particular use-case. Best regards!</p>

**Comment by evilbullet** [comment]
<p>I + to this feature</p>

**Comment by devlinjunker** [comment]
<p>+1 to this feature also, filtering by included/mentioned issues or pulls would be great!</p>

**Comment by HarrisonLovell** [comment]
Was a feature like this implemented?
